### PR TITLE
(feat) Add icon to add sticky notes

### PR DESCRIPTION
**Pull request by usamaidrsk**
# Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. Ensure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing#contributing-guidelines) label (such as `feat`, `fix`, or `chore`, among others). See existing PR titles for inspiration.

## For changes to apps

- [ ] My work conforms to the [**O3 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://om.rs/o3ui).

## If applicable

- [ ] My work includes tests or is validated by existing tests.
- [ ] I have updated the [esm-framework mock](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-framework/mock.tsx) to reflect any API changes I have made.

## Summary

This PR adds new icon - sticky-note-add

### CODE REVIEW

File: packages/framework/esm-styleguide/src/icons/icon-registration.ts [hunk 1]
```diff
@@ -66,6 +66,7 @@ import user from './svgs/user.svg';
 import viewOff from './svgs/view-off.svg';
 import view from './svgs/view.svg';
 import warning from './svgs/warning.svg';
+import stickyNoteAdd from './svgs/sticky-note-add.svg';
```
Comment (ibacher) [inline packages/framework/esm-styleguide/src/icons/icon-registration.ts:69]: Keep this alphabetical please.

File: packages/framework/esm-styleguide/src/icons/icon-registration.ts [hunk 2]
```diff
@@ -137,4 +138,5 @@ export function setupIcons() {
   addSvg('omrs-icon-view-off', viewOff);
   addSvg('omrs-icon-view', view);
   addSvg('omrs-icon-warning', warning);
+  addSvg('omrs-icon-sticky-note-add', stickyNoteAdd);
```
Comment (ibacher) [inline packages/framework/esm-styleguide/src/icons/icon-registration.ts:141]: Ditto here

Comment (ibacher) [reply packages/framework/esm-styleguide/src/icons/icon-registration.ts:141]: Both of these just help ensure we can quickly verify what's been implemented and what hasn't.

File: packages/framework/esm-styleguide/src/icons/svgs/sticky-note-add.svg
Comment (ibacher) [inline packages/framework/esm-styleguide/src/icons/svgs/sticky-note-add.svg:1]: In the SVG remove width, height, viewBox, and fill attributes from all elements.

**Review by ibacher — CHANGES_REQUESTED** [review]
Thanks @usamaidrsk! A few small changes.

File: packages/framework/esm-styleguide/src/icons/icon-registration.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (ibacher) [reply packages/framework/esm-styleguide/src/icons/icon-registration.ts:141]: `stickyNoteAdd` is consistent with how Carbon names things. Specifically, this is `sticky-note--add`, i.e., the `add` variation of the `sticky-note` icon.

**Review by ibacher — APPROVED** [review]

**Comment by denniskigen** [comment]
I wouldn't expect 117 changed files for such a trivial change.

File: packages/framework/esm-framework/docs/classes/OpenmrsFetchError.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ Error.constructor
 
 #### Defined in
 
-[packages/framework/esm-api/src/openmrs-fetch.ts:298](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-api/src/openmrs-fetch.ts#L298)
+[packages/framework/esm-api/src/openmrs-fetch.ts:298](https://github.com/usamaidrsk/openmrs-esm-core/blob/main/packages/framework/esm-api/src/openmrs-fetch.ts#L298)
```
Comment (denniskigen) [inline packages/framework/esm-framework/docs/classes/OpenmrsFetchError.md:56]:
```suggestion
[packages/framework/esm-api/src/openmrs-fetch.ts:298](https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-api/src/openmrs-fetch.ts#L298)
```

Comment (denniskigen) [reply packages/framework/esm-framework/docs/classes/OpenmrsFetchError.md:56]: And elsewhere in many different places.

**Review by denniskigen — CHANGES_REQUESTED** [review]

**Comment by ibacher** [comment]
Oooh... I missed that, but yes, the URL are all incorrect.

**Comment by ibacher** [comment]
One last thing: please add a placeholder to the styleguide's mock.ts file for this.

**Comment by usamaidrsk** [comment]
> One last thing: please add a placeholder to the styleguide's mock.ts file for this.

I have added that. Thanks @ibacher

**Comment by ibacher** [comment]
Should've caught this, but in future we need a ticket to go with any change, other than trivial bugfixes.

**Pull request merged**